### DISCUSSION TITLE
JDK  1.8.0_302 编译项目失败问题

### DISCUSSION THREAD

**Discussion by zck573693104** [apache/streampark#941] [Q&A] [answered]
操作系统版本;
win11
jdk版本：
JDK 1.8.0_302
scala版本：
2.11.12
代码分支：
1.2.2-release
编译命令：
 mvn clean install -DskipTests -Denv=prod
错误详情:
[ERROR] Failed to execute goal net.alchim31.maven:scala-maven-plugin:4.3.0:compile (install-scala) on project streamx-common: Execution install-scala of goal net.alchim31.maven:scala-mav
en-plugin:4.3.0:compile failed: An API incompatibility was encountered while executing net.alchim31.maven:scala-maven-plugin:4.3.0:compile: java.lang.NoSuchMethodError: org.fusesource.ja
nsi.AnsiConsole.wrapOutputStream(Ljava/io/OutputStream;)Ljava/io/OutputStream;

**Comment by bluesky2535** [comment]
同样的问题，兄台解决了没？

**Comment by bluesky2535** [comment, marked as answer]
![image](https://github.com/apache/incubator-streampark/assets/15731203/f585af23-4396-4c23-a8f9-07d2127bc861)
我把这个插件注释掉，编译打包成功了
